### PR TITLE
Port new Icons and PRs from ERIS.

### PR DESCRIPTION
**Pull request by Guidesu**
About The Pull Request
Ports a few icons from eris!
Simply enough.

	
<hr>
</details>

## Changelog
:cl:
add: New sprites from Eris! Janitorial supplies and plants. All credits to the creators! 
delete: old sprites

/:cl:

### CODE REVIEW

**Comment by Guidesu** [comment]
Making a PR to make it easier for me :)
Thanks. Updating images soon.

**Comment by Guidesu** [comment]
image
image
https://private-user-images.githubusercontent.com/61243846/294793903-ead1f944-589d-4af5-90f2-65659dcc6579.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU3Nzg4MzcsIm5iZiI6MTcxNTc3ODUzNywicGF0aCI6Ii82MTI0Mzg0Ni8yOTQ3OTM5MDMtZWFkMWY5NDQtNTg5ZC00YWY1LTkwZjItNjU2NTlkY2M2NTc5LmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE1VDEzMDg1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNkMjllNzEwZTIwNjdlODFmMzY1NDM5MzkwZDgyYjZmYjU0NmMwNzhlYTU3MDRkMjQzNzNmZGM2NTA5YzU4NjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.U4yPjVvRxtpyfGhL_aTtGRxB4PGRDlh1hanVXLLjkX0

**Comment by Trilbyspaceclone** [comment]
You need to change item descs i think for the plants that are changed
Also plant 19 and plant39 are using the same sprite

File: code/game/objects/structures/flora/flora_potted.dm [hunk 1]
```diff
@@ -91,8 +91,8 @@
 	needs_to_maintain = TRUE
 
 /obj/structure/flora/pottedplant/sticky
-	name = "sticky potted plant"
-	desc = "This is an odd plant. Its sticky leaves trap insects."
+	name = "potted sun flower"
+	desc = "This is a sun flower! You feel a urge to sing a silly music about sun flowers. Here comes the sun."
```
Comment (Trilbyspaceclone) [inline code/game/objects/structures/flora/flora_potted.dm:95]: this desc is a bit weird
1 it tells me how i feel

**Comment by Trilbyspaceclone** [comment]
Overgrown plant dosnt match its sprite now?
Its now a rose looking tree

**Pull request merged**